app_flutter/test/utils/wrapper.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:app_flutter/service/google_authentication.dart';
+import 'package:app_flutter/state_provider.dart';
+import 'package:flutter/material.dart';
+
+import 'fake_agent_state.dart';
+import 'fake_flutter_build.dart';
+import 'fake_index_state.dart';
+
+import 'mocks.dart';
+
+class FakeInserter extends StatelessWidget {
```

Comment:
nit: It was unclear to me what this was doing when reviewing the test code. I think something like `FakeApp` is clearer, however, I would expect `FakeApp` to include the `MaterialApp` widget.

Reply:
The FakeInserter inserts the fakes. :-)

I can take a look and see if it makes sense to add MaterialApp to this and call it FakeApp.